codec-memcache/src/main/java/io/netty/handler/codec/memcache/binary/BinaryMemcacheMessage.java

```diff
@@ -160,14 +160,14 @@
      *
      * @return the key of the document.
      */
-    String key();
+    ByteBuf key();
 
     /**
      * Sets the key of the document.
```

Comment:
Should we also add something to the javadocs to tell that once this is called the ownership and so the responsibility to call `ByteBuf.release()` is transferred ?